packages/react-devtools-shared/src/devtools/views/Components/InspectedElementContext.js

```diff
@@ -78,54 +74,44 @@ export function InspectedElementContextController({children}: Props) {
   if (elementHasChanged) {
     setState({
       element,
-      inspectedPaths: {},
+      path: null,
     });
   }
 
   // Don't load a stale element from the backend; it wastes bridge bandwidth.
-  const inspectedElement =
-    !elementHasChanged && element !== null
-      ? inspectElement(
-          element,
-          state.inspectedPaths,
-          forceUpdateRef.current,
-          store,
-          bridge,
-        )
-      : null;
+  let inspectedElement = null;
+  if (!elementHasChanged && element !== null) {
+    inspectedElement = inspectElement(element, state.path, store, bridge);
+  }
 
   const inspectPaths: InspectPathFunction = useCallback<InspectPathFunction>(
     (path: Path) => {
       startTransition(() => {
-        forceUpdateRef.current = true;
```

Comment:
Why is this no longer needed?

Reply:
I guess this is replaced by the `setState` that sets `path` to `null`?

Reply:
The reason it was needed before was b'c the backend wasn't stateful, so if the frontend asked it to hydrate a new path- it might think "this thing hasn't rendered since you last asked, so I'll send the no-change response".